src/test/resources/features/deleteAcount.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Delete an account
+  Background:
+    Given I sign in as "owner"
+    And I click the new project button
+    And I create the project with:
+      | name    | My_Project  |
+      | account | account1    |
+      | privacy | public      |
+    And I have an account created "My Account"
+
+  Scenario: Delete an account
+    When I open the account settings for "My Account"
+    And I delete the account
+    Then I see the confirmation message with the account "My Account"
+    But I don't see the account "My Account" in the accounts menu
+    But I don't see the account "My Account" in the create project form
+    But I don't see the account "My Account" in project settings
```

Comment:
Otra observacion comun que vi
en la validaciones deberian estar explicitamente las navegaciones que se hace en las paginas
basicamente reusar los anteriores steps definitions

por ejemplo:

Then I validate the message is "AccountTest01 was successfully deleted."
And I validate the "AccountTest01" account is not displayed in the Accounts list
And I go to the Dashboard
And I click the new project button
And I validate the "AccountTest01" account is not displayed in the account dropdown
And I create the project with:
| name | MyProject27 |
| account | account1 |
And I click the "More" tab in projects page
And I validate the "AccountTest01" account is not displayed in the change account dropdown.

No necesariamente sera asi
pero ya te da un ejemplo de como deberia ser

Reply:
done